sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableClientTest.java

```diff
@@ -790,6 +794,14 @@ void submitTransactionAsyncWithSameRowKeys() {
             assertTrue(e.getMessage().contains("partitionKey='" + partitionKeyValue));
             assertTrue(e.getMessage().contains("rowKey='" + rowKeyValue));
 
+            return;
+        } catch (TableServiceException e) {
+            assertTrue(IS_COSMOS_TEST);
+            assertTrue(e.getMessage().contains("Status code 400"));
```

Comment:
Checking for exception messages is not reliable. Service can change the error message and these tests will start failing. Are there other fields in the exception that are part of the service contract that we can use to validate?

Reply:
I think we can check for the actual status code for now until we figure out if the Cosmos service not returning a multipart response is intended behavior.